### PR TITLE
Updated S3 documentation for HeadBucket operation

### PR DESCRIPTION
**Pull request by dennisreep**
**This should not be pushed live yet. This should go live when AWS S3 version 3.1.0 is live in the Marketplace.**

@katarzyna-koltun-mx Can you have a look at the changes in this pull request? I added extra information for the HeadBucket operations and besides that I made the text more consistent and flow better (??). Thank you!

### CODE REVIEW

**Comment by MarkvanMents** [comment]
@dennisreep - can this wait until next Tuesday before being reviewed? Kasia is on leave until then.

**Comment by dennisreep** [comment]
> @dennisreep - can this wait until next Tuesday before being reviewed? Kasia is on leave until then.

We can probably wait until then. The code itself needs to be reviewed and released first.

**Comment by MarkvanMents** [comment]
> > @dennisreep - can this wait until next Tuesday before being reviewed? Kasia is on leave until then.
> 
> We can probably wait until then. The code itself needs to be reviewed and released first.

OK - if things progress more quickly, please let me know and I'll pick it up before Kasia returns.

File: content/en/docs/appstore/use-content/modules/aws/aws-s3-connector.md [hunk 1]
```diff
@@ -210,11 +210,24 @@ The entities in the table below describe all generalizations. These are reused b
...
+| Attribute | Description | 
+| --- | --- |
+| `BucketName` | Describes the name of the bucket | 
+
+#### 4.1.18 HeadBucketResponse {#headbucketresponse}
+
+| Attribute | Description | 
+| --- | --- |
+| `Region` | Describes the AWS location of the given bucket |
+| `AccessPointAlias` | Describes wether the bucket was requested through an Access Point Alias |
```
Comment (strategicalliances) [inline content/en/docs/appstore/use-content/modules/aws/aws-s3-connector.md]: it should be **whether**

**Review by katarzyna-koltun-mx — APPROVED** [review]

**Comment by katarzyna-koltun-mx** [comment]
Approved and waiting for the go ahead to publish.

**Pull request merged**